src/test/java/net/sf/jabref/logic/layout/LayoutTest.java

```diff
@@ -86,15 +82,6 @@ public void testHTMLChar() throws IOException {
     }
 
     @Test
-    @Ignore
```

Comment:
If that's really what we want to achieve here \n\n -> ' ' 
I'm not sure about this that's why I removed the test.